ui/src/keyboardMappings.ts

```diff
@@ -43,7 +43,7 @@ export const keys = {
   F13: 0x68,
   Home: 0x4a,
   Insert: 0x49,
-  IntlBackslash: 0x31,
+  IntlBackslash: 0x64,
```

Comment:
I was going to correct that when I was doing the cleanup/refactor and forgot. Thanks!

Reply:
Thanks for the review!